lecture/forms.py

```
@@ -1,17 +1,26 @@
 from django import forms
-from .models import Lecture, LecturePDF
-
+from .models import Lecture, LecturePDF,LectureVideo
 
 class LectureForm(forms.ModelForm):
     class Meta:
         model = Lecture
-        fields = ['title', 'course']
-
+        fields = ['title']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)